libdnf5/rpm/package_query.cpp

```diff
@@ -1943,12 +1945,24 @@ void PackageQuery::PQImpl::filter_nevra(
             filter_nevra_internal_str<cmp_lte>(pool, c_pattern, sorted_solvables, filter_result);
             break;
         case libdnf5::sack::QueryCmp::GLOB:
-            filter_glob_internal<&libdnf5::solv::RpmPool::get_nevra>(
-                pool, c_pattern, *pkg_set.p_impl, filter_result, 0);
+            found = pattern.find(':');
```

Comment:
Nit: I think it would be more readable to declare `found` down here instead of above the switch

Reply:
Fixed